howtos/documentation-guidelines.md

```diff
@@ -48,6 +52,10 @@ Specific Optimize versions are aligned with Camunda versions as follows:
 | 8.0             | 3.8.0            |
 | 1.3             | 3.7.0            |
 
+### Interactive API Explorer versions
```

Comment:
@pepopowitz should there be a table of version associations for the interactive API explorer as well?

Reply:
I don't think so! 

The table under "version alignment" shows how main docs versions align with Optimize versions. 

The interactive API explorers will only ever have 1 version, because they're based on the API versions.

Reply:
Oh wait, I already have other changes from Christina to make! I'll include this change, too.